### PR TITLE
Notes 100: Comments on notes

### PR DESCRIPTION
**Pull request by OlegMoshkovich**
#978

This PR includes the following changes:

- Added a "Leave Comment" note card to the individual note view.
- Linked the "Add Comment" button to navigate to the individual note view, where the add comment note card is visible.
- Added an information component to notify the user that they need to log in to leave a comment.
- Added an information component to indicate if the note is locked and no comments can be created.

## Tests
[percy](https://percy.io/8fe2b2f1/share/builds/34123335)

[percy - note-contains-link-to-github-issue.cy.js](https://percy.io/8fe2b2f1/share/builds/34178600) - New golden with wait for model animation

### CODE REVIEW

**Comment by pablo-mayrgundter** [comment]
Isn't this broken?  Looks like a note has been selected, so expect to see comments
image

File: cypress/e2e/notes-100/comments-on-a-note.cy.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import '@percy/cypress'
+import {
+  homepageSetup,
+  returningUserVisitsHomepageWaitForModel,
+  auth0Login,
+} from '../../support/utils'
+
+/** {@link https://github.com/orgs/bldrs-ai/projects/50/views/1?pane=issue&itemId=52262146} */
```
Comment (pablo-mayrgundter) [inline cypress/e2e/notes-100/comments-on-a-note.cy.js:8]: Link to issue directly instead of view in project

File: src/Components/Notes/Notes.jsx [hunk 2]
```diff
@@ -116,18 +117,29 @@ export default function Notes() {
...
+          synched={selectedNote.synched}
+          title={selectedNote.title}
+          username={selectedNote.username}
+        />
+      }
+      <ListItem key={'commentCreate'}>
+        {user && selectedNote && !selectedNote.locked && <NoteCardCreate isNote={false} noteNumber={selectedNote.number}/>}
+      </ListItem>
+      {selectedNote && !user &&
+        <Box sx={{paddingBottom: '10px'}}>
```
Comment (pablo-mayrgundter) [inline src/Components/Notes/Notes.jsx:137]: em instead of px.. 0.5 or 1em?

**Review by pablo-mayrgundter — APPROVED** [review]

File: src/Components/Notes/Notes.jsx (hunk 2 above)
Comment (OlegMoshkovich) [reply src/Components/Notes/Notes.jsx:137]: yep

**Comment by OlegMoshkovich** [comment]
you are right about Percy snapshot, will take a closer look 🙏

File: cypress/e2e/notes-100/comments-on-a-note.cy.js (hunk 1 above)
Comment (OlegMoshkovich) [reply cypress/e2e/notes-100/comments-on-a-note.cy.js:8]: done

**Pull request merged**